packages/subiquity_client/lib/src/types.dart

```diff
@@ -5,6 +5,36 @@ part 'types.g.dart';
 
 // ignore_for_file: invalid_annotation_target
 
+// subiquity serializes `Dict[int, Any]` as `List[List[int, Any]]`
+class JsonMapConverter
```

Comment:
I chose a more generator-friendly and property-agnostic syntax for the `Map<int, String>` JSON conversion workaround.